hiro.pb.go

```diff
@@ -14,8 +14,8 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.34.2
-// 	protoc        v5.27.1
+// 	protoc-gen-go v1.26.0
```

Comment:
Try updating your `protoc-gen-go` plugin, I think the older version is causing a larger than needed diff here.

Reply:
Sorry about that, not sure why it didn't have the latest version